homeassistant/components/panel_iframe.py

```diff
@@ -8,22 +8,28 @@
 
 import voluptuous as vol
 
+from homeassistant.const import (CONF_ICON, CONF_URL)
 import homeassistant.helpers.config_validation as cv
 
-DOMAIN = 'panel_iframe'
 DEPENDENCIES = ['frontend']
 
+DOMAIN = 'panel_iframe'
+
 CONF_TITLE = 'title'
-CONF_ICON = 'icon'
-CONF_URL = 'url'
+
+CONF_ABSOLUTE_PATH_ERROR_MSG = "Invalid absolute path in relative URL"
+CONF_ABSOLUTE_PATH_REGEX = r'\A/'
```

Comment:
This is an incorrect const name. It's checking if it's a relative url, not an absolute url.

Reply:
Thank you for your feedback. I used the term "ABSOLUTE_PATH" because we are checking for an absolute path in a relative URL. I'll not discuss with you - the creator of this great piece of software and master chief of a great community. Please check it again if this explanation makes sense to you, give "thumbs down" and I'll change.

Reply:
We should use the term Relative Url because we check if the passed value is a valid relative URL. If not, show an error message that it's not a valid relative URL.

The check for being an absolute URL is done by `cv.Url`. `vol.Any` will make sure both checks are performed and raise if none pass.

Reply:
No need to copy paste the same answer on every comment.

Reply:
Got it and checked in the requested changes. Changed also the PR #11808